x-pack/filebeat/input/awscloudwatch/_meta/terraform/.terraform.lock.hcl

```diff
@@ -0,0 +1,57 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
I have the impression this file should not be versioned

Reply:
I thought about this too 😬 This file is not included in gitignore in `aws-s3` input and also I was reading https://stackoverflow.com/questions/67963719/should-terraform-lock-hcl-be-included-in-the-gitignore-file online. Maybe we should keep it versioned.

Reply:
:+1: on keeping it versioned. This allow us to control provider versions upgrade and help with environment reproducibility (especially with CI).

Reply:
Will do, I will resolve this conversataion then. Thanks for the inputs!